examples/website/plot/plot-layer/label-vertex.glsl.ts

```diff
@@ -27,22 +27,19 @@ in vec3 normals;
 in vec2 texCoords;
 in vec2 instancePositions;
 in vec3 instanceNormals;
-in float instanceIsTitle;
+in float instanceOffsets;
```

Comment:
`instanceIsTitle` was used and only used to set offsets, I thought this would be a little easier to read.

Overall I'm not trying to give this layer a ton of functionalities not needed by the website. However I do want to implement it in a way that demonstrates best practices.